cmd/umcli/accounts.go

```diff
@@ -0,0 +1,204 @@
+// Command umcli provides admin command line tool to manipulate accounts with admin rights.
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/lvl484/user-manager/logger"
+	"github.com/lvl484/user-manager/model"
+)
+
+// StatusAccount structure that hold responce of User Manager client subservice
+type StatusAccount struct {
+	code    int
+	message string
+}
+
+var (
+	StatusInfoOK             = StatusAccount{code: 200, message: "An account info"}
+	StatusCreateOK           = StatusAccount{code: 201, message: "Successfully created"}
+	StatusUpdateOK           = StatusAccount{code: 200, message: "Successfully updated"}
+	StatusDeleteOK           = StatusAccount{code: 204, message: "Successfully deleted"}
+	StatusBadRequest         = StatusAccount{code: 400, message: "Bad request"}
+	StatusAuthenticateFailed = StatusAccount{code: 401, message: "Authenticate failed"}
+	StatusAccountNotExist    = StatusAccount{code: 404, message: "Account does not exist"}
+	StatusLoginInUse         = StatusAccount{code: 409, message: "Login in use"}
+	StatusUnexpectedError    = StatusAccount{code: 444, message: "Unexpected error"}
+)
```

Comment:
How about to change 200, 201 etc. to http.StatusOK and other?

Reply:
That answers are taken from our swagger-api.yaml